src/Utopia/Messaging/Adapters/SMS/ClickSend.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+// Reference Material
+// https://developers.clicksend.com/docs/rest/v3/#send-sms
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+class ClickSend extends SMSAdapter
+{
+    /**
+     * @param  string  $username ClickSend Username
+     * @param  string  $apiKey ClickSend API Key
+     */
+    public function __construct(
+        private string $username,
+        private string $apiKey
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return 'ClickSend';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1;
```

Comment:
Please find the correct value for this, the `process` function looks like it allows sending to multiple numbers at once but if this value is 1, an exception will be thrown if `$message->getTo()` has more than 1 value

Reply:
Done. I had to do some rewriting to make it work.